manifests/forward-to-fluentd/fluentd-cluster-cfg-output-es.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: fluentd.fluent.io/v1alpha1
+kind: Fluentd
+metadata:
+  name: fluentd-forward
+  namespace: kubesphere-logging-system
+  labels:
+    app.kubernetes.io/name: fluentd
+spec:
+  globalInputs:
+    - type: forward
+      forward: 
+        bind: 0.0.0.0
+        port: 24224
+  replicas: 1
+  image: zhu733756/fluentd:config-reloader
+  fluentdCfgSelector: 
+    matchLabels:
+      config.fluentd.fluent.io/enabled: "true"
+   
+---
+apiVersion: fluentd.fluent.io/v1alpha1
+kind: ClusterFluentdConfig
+metadata:
+  name: fluentd-config
+  labels:
+    config.fluentd.fluent.io/enabled: "true"
+spec:
+  watchedNamespaces: 
+    - kube-system
+    - kubesphere-logging-system
+    - kubesphere-monitoring-system
+  outputSelector:
+    matchLabels:
+      output.fluentd.fluent.io/enabled: "true"
+
+---
+apiVersion: fluentd.fluent.io/v1alpha1
+kind: ClusterOutput
+metadata:
+  name: fluentd-stdout
+  labels:
+    output.fluentd.fluent.io/enabled: "true"
+spec: 
+  outputs: 
```

Comment:
we might need to consider if the `outputs:` layer is necessary and if the `type` field could be removed。

Reply:
This part will fix after I come back from the holidays. :)

Reply:
What I think is that the `outputs` field is somehow duplicated with the CRD name, maybe we can change it to `items`?
The same applies to `filters`.

Your original idea of using an array to store outputs or filters is good, we can keep using the array.
Let's see if it's possible to deal with the type part, it's also ok to keep current implementation if we cannot find a better way

Reply:
I think the type field combined into the plugin is reasonable, I will try to fix it.